src/sentry/api/endpoints/team_members.py

```diff
@@ -3,23 +3,54 @@
 from rest_framework.response import Response
 
 from sentry.api.bases.team import TeamEndpoint
-from sentry.api.serializers import serialize
-from sentry.models import InviteStatus, OrganizationMember
+from sentry.api.serializers import Serializer, register, serialize
+from sentry.models import InviteStatus
+from sentry.models.organizationmemberteam import OrganizationMemberTeam
+
+
+@register(OrganizationMemberTeam)
+class DetailedOrganizationMemberTeamSerializer(Serializer):
+    def __init__(self, *args, **kwargs):
+        self.team = kwargs.pop("team", None)
+        super().__init__(*args, **kwargs)
+
+    def get_attrs(self, item_list, user, **kwargs):
+        attrs = {}
+        for org_member_team in item_list:
+            attrs[org_member_team] = {
+                "org_member": org_member_team.organizationmember,
+                "user": org_member_team.organizationmember.user,
+            }
+        return attrs
+
+    def serialize(self, obj, attrs, user):
+        org_member = serialize(attrs["org_member"])
+        org_member["user"] = serialize(attrs["user"])
```

Comment:
I'd probably recommend that you serialize these in bulk in `get_attrs`. Also, `OrganizationMemberSerializer` already includes the user, so we probably don't need to also serialize the user